annotate.el

```diff
@@ -2389,6 +2389,24 @@ pressed."
         (annotate-update-visited-buffer-maybe filename)
         (annotate-show-annotation-summary query nil nil)))))
 
+(cl-defun annotate-wrap-text (text &optional (wrapper "\""))
+  (concat wrapper text wrapper))
+
+(cl-defun annotate-unwrap-text (text &optional (wrapper "\""))
```

Comment:
I wonder if a solution using `substring` wouldn't be cleaner. Regular expressions seem a bit heavy handed for this job.

Reply:
Hi!

I agree that using regex is too much and, moreover, makes the code a bit ugly (in my opinion); i pushed a version that uses substring.

Bye!
C.